tests/test_other.py

```diff
@@ -7926,6 +7926,11 @@ def test_wasm_backend(self):
     if old == '1': return # already the default
     try:
       os.environ['EMCC_WASM_BACKEND'] = '1'
+      if EMCC_SKIP_SANITY_CHECK_NAME in os.environ:
+        # Cached sanity check result was for different configuration
+        del os.environ[EMCC_SKIP_SANITY_CHECK_NAME]
```

Comment:
it's usually better to skip the test with a message than to modify the environment.

in any case, though, I don't think we need to check sanity here?

Reply:
As I understand, we already skip this test if no environment modification (that was in place before this PR) is required. :) As you already noted, we don't need to perform checks before every test case. I redo sanity check **for this particular test case** because here we **artificially changed** the condition on which sanity check control flow depends (in fact, I don't handle very well another input: the `force` function argument here and in the other places).